src/AppBundle/Resources/translations/odr-bank-accounts.en.yml

```diff
@@ -9,6 +9,22 @@ startPage:
     accounts, savings accounts and ISAs.
   pageSectionDescription3: |
     We will also need to know the balance of each account.
+  hiddenText01:
+    summary: More on how to carry out your duties 
+    content:
+      para01: "As a property and affairs deputy, you must now be sure to keep copies of:"
+      listItem01: bank statements
+      listItem02: contracts for services or tradespeople
+      listItem03: receipts
+      listItem04: letters and emails about your activities as a deputy
+      para02: |
+        You must include details of how you've managed %client%'s money in your 
```

Comment:
I don't see why we can't have that HTML in the YML file. It's better to retain the URL in the YML file than to have it in the view.